README.md

```diff
@@ -167,89 +301,139 @@ env:
     - PYTHON_LINT_OPTIONS="source --verbose"
 ```
 
-### shared_configs/python-tester.yml
+#### shared_configs/python-tester-conda.yml
+`python-tester-conda.yml` runs any pytest tests it finds after installing the
+specified requirements via conda.
+The current test matrix includes Python 3.6, 3.7 and 3.8.
+This job uses the workspaces named `conda` for the `noarch` package built
+before by the `anaconda-build.yml`.
+
+##### usage:
+This configuration can be added to the `test` stage of a Travis build by
+importing it in your `.travis.yml`:
+``` yaml
+import:
+  - pcdshub/pcds-ci-helpers:travis/shared_configs/python-tester-conda.yml
+```
+##### arguments:
+Test requirements files can be specified by assigning values to
+`CONDA_REQUIREMENTS`. `CONDA_REQUIREMENTS` defaults to `dev-requirements.txt`.
+If no requirements are necessary, the file can be blank.
+Additional test dependencies can be specified via the `CONDA_EXTRAS` variable.
+The package name must be informed via the `CONDA_PACKAGE` variable.
+
+Additional dependencies not specified at the requirements files above can be
+passed to the install process via the `PIP_EXTRAS` variable.
+
+``` yaml
+env:
+  global:
+    # The name of the conda package
+    - CONDA_PACKAGE="my_package"
+    # The folder containing the conda recipe (meta.yaml)
+    - CONDA_RECIPE_FOLDER="conda-recipe"
+    # Extra dependencies needed to run the tests which are not included
+    # at the recipe and dev-requirements.txt. E.g. PyQt
+    - CONDA_EXTRAS="pip pyqt=5 happi"
+    # Requirements file with contents for tests dependencies
+    - CONDA_REQUIREMENTS="dev-requirements.txt"
+```
+
+#### shared_configs/python-tester-pip.yml
 `python-tester.yml` runs any pytest tests it finds after installing the
-specified requirements.
+specified requirements via PIP using Python 3.6.
 
-#### usage:
+##### usage:
 This configuration can be added to the `test` stage of a Travis build by
 importing it in your `.travis.yml`:
 ``` yaml
 import:
-  - pcdshub/pcds-ci-helpers:travis/shared_configs/python-tester.yml
+  - pcdshub/pcds-ci-helpers:travis/shared_configs/python-tester-pip.yml
 ```
-#### arguments:
+##### arguments:
 Requirements files can be specified by assigning values to `REQUIREMENTS` and
 `DEV_REQUIREMENTS`. `REQUIREMENTS` defaults to `requirements.txt` and
 `DEV_REQUIREMENTS` defaults to `dev-requirements.txt`. If no requirements are
 necessary, the file can be blank.
+
+Additional dependencies not specified at the requirements files above can be
+passed to the install process via the `PIP_EXTRAS` variable.
+
 ``` yaml
 env:
   global:
     - REQUIREMENTS: requirements.txt
     - DEV_REQUIREMENTS: dev-requirements.txt
+    - PIP_EXTRAS="PyQt5 happi"
 ```
 
-### shared_configs/doctr-upload.yml
+#### shared_configs/docs-build.yml
 `docs-build.yml` runs through the build of the package's documentation to
 ensure it works properly.
+After building the documentation it is available via the `workspaces` named
```

Comment:
```suggestion
After building the documentation it is available via the workspace named
```
'workspaces' was unquoted above so I'm changing it here for consistency. If you'd rather have it quoted, change the instance in python-tester-conda.yml instead.